woocommerce/storefront
# Move Storefront admin notice content to the Storefront Appearance page.

Fixes #1539


We were requested to move the Storefront admin notice to the inbox messages.
Scanning for `admin_notice` has shown that Storefront uses only one notices. This is the main welcome "giant" notice:
![image](https://user-images.githubusercontent.com/17271089/100101064-0924a000-2e62-11eb-84b2-0b08c5b112fa.png)

I have decided to preserve the old `admin_notices` functionality for setups that use WooCommerce versions < 4.6.0.
This version was chosen because from this version the new WooCommerce home screen that was introduced in WooCoomerce 4.3.0 has become the default selection.





### Screenshots


From WooCommerce->Home:

![image](https://user-images.githubusercontent.com/17271089/100101294-56a10d00-2e62-11eb-9379-57a96ffd1d7c.png)

From other WooCommerce submenus:

![image](https://user-images.githubusercontent.com/17271089/100101446-86e8ab80-2e62-11eb-90e5-9dae172653bf.png)


### How to test the changes in this Pull Request:



1. Use WooCommerce 4.6.0+
2. On a fresh site ( or with wiped storefront options ) install and activate this Storefront Branch
3. Observe WooCommerce->Home or other WooCommerce submenus inbox area.
4. Click on the `Let's go!` button it should take you to the customizer like previously. It should behave like if the `Apply storefront homepage template` and `Add example products` would be unchecked/

This implementation is missing the homepage template and the example products feature. We may be in the scenario where we don't actually need this because wc-admin setup list task `Personalize my store` is doing that so doubling this feature would be counterproductive. The discussion is taking place here: pbIJXs-s6-p2
The inbox messages should not be used the way we used our admin_notice. We can't add a form that would host our checkboxes. One of the options I see is to use 3 inbox messages that would exist as separate actions. This sounds reasonable because the inbox is designed to accommodate multiple messages. Although I think it is OK this approach needs design guiding. 
I have tested how it would look for two:
![image](https://user-images.githubusercontent.com/17271089/100104179-23607d00-2e66-11eb-8129-ea5c1adf8bf5.png)
The Add products button is not connected, it is just a mockup, but it shows how this approach would look.

If using multiple messages is not an option we would need to take a look at the tasks list direction.

@haszari thoughts?

@LevinMedia If we would go one message or multiple messages options, could you please help with copy and overall design. There is little flexibility with Inbox Messages so probably copy only, or we can have a banner :P 




### Changelog

Move Storefront welcome notice to the WooCommerce Admin inbox messages.
